src/extensions/mountainview/MVSortingView/MVSortingView.tsx

```diff
@@ -212,6 +213,11 @@ const MVSortingView: FunctionComponent<SortingViewProps> = (props) => {
                             selectionDispatch={props.selectionDispatch}
                         />
                     </Expandable>
+
+                    {/* Options */}
```

Comment:
A new options box in the left control panel allowing toggle between geom and vertical modes